### PR TITLE
A quick pass of corrections and formatting.

### PR DESCRIPTION
**Pull request by awholegunch**
Fix for https://issues.cask.co/browse/CDAP-4877

A quick pass over the documentation to fix formatting and minor errors.
Standardized on two blank lines before each title and no blank line after a title, to improve readability.

Note: there are two Database-batchsink.md and two Database-batchsource.md:
- core-plugins/docs/Database-batchsink.md
- database-plugins/docs/Database-batchsink.md

and
- core-plugins/docs/Database-batchsource.md
- database-plugins/docs/Database-batchsource.md

Which ones should be deleted?

### CODE REVIEW

**Comment by albertshau** [comment]
The database ones in core-plugins should be deleted

**Comment by albertshau** [comment]
lgtm though there are conflicts

**Comment by awholegunch** [comment]
I have resolved the conflict, but please review the last two commits closely.

**Comment by albertshau** [comment]
@gokulavasan please take a look at the last commit for the teradata source.  Otherwise lgtm

File: hbase-plugins/docs/HBase-batchsink.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
 Use Case
 --------
-
 The sink is used whenever you need to write to an HBase table in batch. For example,
 you may want to periodically dump the contents of a relational database into an HBase table.
 
+
 Properties
 ----------
 **tableName:** The name of the table to read from.
```
Comment (gsps1) [inline hbase-plugins/docs/HBase-batchsink.md]: there is an issue here, this should be table to write to.. this seems to be copied from batchsource.

Comment (gsps1) [inline hbase-plugins/docs/HBase-batchsink.md]: also please add a note, saying this table should already exist before running the pipeline.

**Comment by awholegunch** [comment]
Please look at the changes in the last commit, as I have addressed both @gokulavasan and some other errors that I found.

**Comment by albertshau** [comment]
@gokulavasan still hasn't looked at the teradata changes. Otherwise lgtm

**Comment by albertshau** [comment]
o nevermind those changes were pulled in from release/1.2 so i assume they don't need to be reviewed.  lgtm

**Comment by gokulavasan** [comment]
LGTM :+1:

**Pull request merged**